src/Arduino_MCHPTouch.h

```diff
@@ -23,6 +23,7 @@ class QtouchClass
   int available();
   int read(int sensor_id);
   void setSensorsSensitivity(unsigned int newSens);
+  void setSensorsSensitivityChannel(unsigned int newSens ,unsigned int btn_channel);
```

Comment:
```suggestion
  void setSensorsSensitivity(unsigned int newSens ,unsigned int btn_channel);
```

Instead of exposing the underlying API which is not too friendly I would recomend to use the existing method name with a different signature. The "channel" name is not currently used in Arduino/Oplà documentation and may be confusing :)